apps/approval/views.py

```diff
@@ -70,6 +68,20 @@ def create_fos_application(request):
     raise Http404
 
 
+def get_expiry_date(started_year, length_of_fos):
+    # Expiry dates should be 15th September, so that we have time to get new lists from NTNU
+    new_expiry_date = datetime.date(
+        started_year, 9, 16) + datetime.timedelta(days=365*length_of_fos)
+    # Expiry dates in the past sets the expiry date to next september
+    if new_expiry_date < datetime.date.today():
```

Comment:
https://docs.djangoproject.com/en/1.9/topics/i18n/timezones/#naive-and-aware-datetime-objects

This matters mostly for datetime objects, so you're right in the fact that it's less important for date. Anywho, it's better practice to do `today = timzone.now().date.today()` and work with `today` rather than doing `datetime.date.today()` every time you need the variable (the same goes for `datetime.date.today()` for that matter). :-)

So yes, I was a bit too quick to respond in this case.